### PR TITLE
load publish profile sqlcmd "value" tag and throw error if not loaded correctly

### PR DESCRIPTION
**Pull request by udeeshagautam**
This PR fixes loading publish profile sqlcmd "value" tag and throws error if not loaded correctly

### CODE REVIEW

File: extensions/sql-database-projects/src/common/utils.ts [hunk 1]
```diff
@@ -120,20 +120,3 @@ export function convertSlashesForSqlProj(filePath: string): string {
 	const parts = filePath.split('/');
 	return parts.join('\\');
 }
-
-/**
- * Read SQLCMD variables from xmlDoc and return them
- * @param xmlDoc xml doc to read SQLCMD variables from. Format must be the same that sqlproj and publish profiles use
- */
-export function readSqlCmdVariables(xmlDoc: any): Record<string, string> {
```
Comment (kisantia) [inline extensions/sql-database-projects/src/common/utils.ts:128]: doesn't project.ts use this to read the variables in the .sqlproj?

Comment (udeeshagautam) [reply extensions/sql-database-projects/src/common/utils.ts:128]: no actually nothing uses it. Project uses the one from profile - I can move it here and remove the profile one

Comment (udeeshagautam) [reply extensions/sql-database-projects/src/common/utils.ts:128]: moved to util removed from publishprofile

Comment (kisantia) [reply extensions/sql-database-projects/src/common/utils.ts:128]: ah ok oops I must've forgotten to move it when moving the publish profile stuff to its own file. Thanks for cleaning up!

**Pull request merged**